pkg/api/v1/deployment.go

```diff
@@ -132,6 +144,44 @@ func (g *DeploymentGroup) StopAllActiveDeployments(ctx echo.Context) error {
 	return g.stopDeployments(deployments, ctx)
 }
 
+func (g *DeploymentGroup) DeleteDeployment(ctx echo.Context) error {
+	cc, ok := ctx.(*auth.HttpAuthContext)
+	if !ok {
+		return echo.NewHTTPError(http.StatusInternalServerError, "Failed to get auth info")
```

Comment:
Is this needed? I thought we took care of this [here](https://github.com/beam-cloud/beta9/blob/main/pkg/auth/http.go#L83-L98).

Reply:
yea you are right